CONTRIBUTING.md

```diff
@@ -19,14 +19,15 @@ Nejdřív zmíníme nějaké pojmy, které se v tomto dokumentu budou objevovat,
 - [HTML](https://en.wikipedia.org/wiki/HTML) - HTML je značkovací jazyk, kterým se popisuje struktura webu. Tj. co jsou nadpisy, odkazy, obrázky, atd.
-- [NodeJS](https://nodejs.org/) - NodeJS je software, co spouští programy v jazyce JavaScript. A v tom je náš web napsaný. Respektive v TypeScriptu, ale o tom později. Při instalaci na Windows nezapomeň zaškrtnout u Zahrnutí v cestě PATH (Add to PATH) možnost "Entire feature will be installed to local hard drive".  
-*Pozor: Aktuální verze (18) není kompatibilní s naším webem (ten používá verzi 14), takže buď najdi starší verzi, nebo se netrap s nějakou instalací a využij spuštění v codespaces, kde se NodeJS automaticky nainstaluje, viz sekce [Nastavení vývojového prostředí](#nastavení-vývojového-prostředí)*
+- [NodeJS](https://nodejs.org/) - NodeJS je software, co spouští programy v jazyce JavaScript. A v tom je náš web napsaný. Respektive v TypeScriptu, ale o tom později. Při instalaci na Windows nezapomeň zaškrtnout u Zahrnutí v cestě PATH (Add to PATH) možnost "Entire feature will be installed to local hard drive".
 - [SCSS](https://sass-lang.com/) - CSS je jazyk, kterým se popisuje vzhled webu. Tj. jaké barvy apod. mají nadpisy, odkazy, obrázky, atd. My používáme jeho rozšíření SCSS.
-- [TypeScript](https://www.typescriptlang.org/) - TypeScript je programovací jazyk, který se překládá do JavaScriptu. Je to jazyk, který má všechny výhody JavaScriptu, ale je o něco přehlednější a příjemnější k psaní. Používá se pro programování logiky webu, tedy jak co bude fungiovat.
+- [TypeScript](https://www.typescriptlang.org/) - TypeScript je programovací jazyk, který se překládá do JavaScriptu. Je to jazyk, který má všechny výhody JavaScriptu, ale je o něco přehlednější a příjemnější k psaní. Používá se pro programování logiky webu, tedy jak co bude fungovat.
 - [Visual Studio Code](https://code.visualstudio.com/) - Editorů kódu je spousta. My doporučujeme Visual Studio Code. Je zdarma, má dobrou podporu pro technologie, co na našem webu požíváme, a má přímo v sobě hezké ovládání nástroje na verzování.
 
 ## Nastavení vývojového prostředí
 
+Způsobů, jak si nastavit vývojové prostředí, je více. První možnost je stáhnout si celý web do svého počítače, pracovat s ním lokálně a změny synchronizovat na github. Další možností je vývoj v [Dockeru](https://cs.wikipedia.org/wiki/Docker) - to je taková light verze virtuálního počítače (tahle možnost bude pravděpodobně nejméně náchylná na problémy „někdo něco změnil a nejde mi spustit vývoj“, protože jí používá většina těch, co takové změny umí udělat:-)). Třetí možnost je nejjednodušší nastavení, protože se stačí v prohlížeči prokliknout do Codespaces a můžeš začít pracovat, ale zase třeba potřebné balíčky si musíš stáhnout pokaždé znova. Ještě existuje možnost rozjet si Docker na Bošánovském serveru, ale tahle možnost dělala takové problémy, že tu radši ani není uvedená.
```

Comment:
To smazání codespace se dá zrušit, pak nemusíš znovu instalovat ty balíčky (bude asi stát víc peněz, ale ne nějak extra moc):

![image](https://github.com/bosancz/bosan.cz/assets/1273865/95b54e6a-f887-43c4-a8be-7db74020955d)

Reply:
To mi přijde zbytečné. Zas tak náročný to není tam nadatlit ty dva příkazy:-)